dataeng/jobs/createJobsNew.groovy

```diff
@@ -50,8 +51,11 @@ for (task in taskMap) {
     catch (Exception e) {
         out.println("File $extraVarsFileName does not exist.")
     }
-
-    task.value(this, commonConfigMap + extraVars)
+    if (task.value instanceof List) {
+        task.value.forEach(this, commonConfigMap + extraVars)
+    } else {
+        task.value(this, commonConfigMap + extraVars)
+    }
```

Comment:
I think you should continue to follow the pattern here of mapping the key to just one Job object. In our meeting, we discussed refactoring the job dsl to iterate over a config map to generate n-many Jenkins jobs from one DSL file